### PR TITLE
Fix Files UI Tests

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: none
release note: none

### CODE REVIEW

File: rn/Teacher/ios/TeacherUITests/Files/FilesUITests.swift [hunk 1]
```diff
@@ -156,6 +156,10 @@ class FilesUITests: MiniCanvasUITestCase {
         FileEditItem.delete.tap()
         app.find(label: "Delete").tap()
 
+        // FIXME: These should have happened automatically.
```
Comment (cobbal) [inline rn/Teacher/ios/TeacherUITests/Files/FilesUITests.swift:159]: Is there a ticket for this?

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/9nw3roi) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F9nw3roi)
teacher: [Link](https://rebrand.ly/a9tm2th) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fa9tm2th)
parent: [Link](https://rebrand.ly/bq8du3n) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fbq8du3n)

File: rn/Teacher/ios/TeacherUITests/Files/FilesUITests.swift (hunk 1 above)
Comment (vanwagonet) [reply rn/Teacher/ios/TeacherUITests/Files/FilesUITests.swift:159]: https://instructure.atlassian.net/browse/MBL-14496

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.24% | 93.24% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**